src/Clap-Tests.package/ClapCommandTest.class/instance/testMatchesWithSingleFlag.st

```diff
@@ -1,14 +1,14 @@
 tests
 testMatchesWithSingleFlag
 	| match |
-	subject addFlag: (ClapFlag withName: 'bar').
+	subject addFlag: ((ClapFlag withName: 'bar') longForm: 'bar').
```

Comment:
we could define a `withLongForm` message that uses the name and calls `longForm:`